source/simulator/stokes_matrix_free.cc

```diff
@@ -1492,7 +1495,7 @@ namespace aspect
                   // of the evaluated viscosity on the active level.
                   for (unsigned int q=0; q<n_q_points; ++q)
                     active_cell_data.viscosity(cell, q)[i]
-                      = std::min(std::max(values_on_quad[q], min_el), max_el);
+                      = std::min(std::max(values_on_quad[q], viscosity_min), viscosity_max);
```

Comment:
Was this the bug before? Did we limit the viscosity only to the local minimum and maximum? But I do not understand why this had caused the issue, limiting to the local minimum and maximum should only be stricter than limiting to the global minimum/maximum. Can you explain what the problem was?